README.md

```diff
@@ -213,6 +213,30 @@ Instead, each directive may contain these arguments:
   * `:compression` (Integer) : The left number of the compression ratio `n:1`, e.g. 12 will apply 12:1 compression. Default: 10.
   * `:tile_size` (Integer) : Pixel dimension of the tiles. Default: 1024
 
+# Development Environment
+
+## Dependencies
+
+* [ImageMagick](https://www.imagemagick.org)
+    * On a mac, do `brew install imagemagick --with-little-cms2 --with-openjpeg`
```

Comment:
As of [Homebrew 2.0.0](https://brew.sh/2019/02/02/homebrew-2.0.0/), homebrew-core accepts no options and uses both of these by default.